C-Plus-Plus/dp/Count_subset_given_difference.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+This problem is solved using dp.
+In this problem we have to count whether two subset difference is equal to sum or not
+if equal then we will return true else we will return false.
```

Comment:
Give a detailed description for the problem

Reply:
done